storage/range.go

```diff
@@ -686,8 +687,8 @@ func (r *Range) addWriteCmd(args proto.Request, reply proto.Response, wait bool)
 		// If the original client didn't wait (e.g. resolve write intent),
 		// log execution errors so they're surfaced somewhere.
 		if !wait && err != nil {
-			log.Warningf("non-synchronous execution of %s with %+v failed: %s",
-				args.Method(), args, err)
+			log.CtxWarningf(ctx.Add(logfield.Error, err),
```

Comment:
interesting, I didn't see that anywhere. Will probably not matter much unless we really bloat our context, which we shouldn't be doing anyways, and a bunch of our vars have different types anyways (NodeID, StoreID, RaftID, error, ...)